resources/platforms.json

```diff
@@ -4477,6 +4477,54 @@
       "standard": true,
       "inherits": "SymbianOS"
     },
+    "Series30": {
+      "match": "*Series40*",
+      "lite": false,
+      "standard": true,
+      "properties": {
+        "Platform": "Series30",
+        "Platform_Maker": "Nokia",
+        "Platform_Description": "Series 30",
+        "Platform_Version": "unknown",
+        "Win16": "false",
+        "Win32": "false",
+        "Win64": "false",
+        "Browser_Bits": "32",
+        "Platform_Bits": "32"
+      }
+    },
+    "Series40": {
```

Comment:
All Nokia devices were detecting as either "JAVA" or "SymbianOS", but I figured that these platforms were more indicative of what the phones actually run.

I didn't bother adding `Series60` or `Series80` since they're UI frameworks on top of SymbianOS.